src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -296,10 +303,34 @@ private static ServerMessage moveCheatingPlayer(ClientMessage message, WebSocket
     private static ServerMessage gameState(int gameId, WebSocketPlayer player) {
         Game.GameState gameState = Game.getGameState(gameId, player);
 
-        return ServerMessage.builder()
-                .messagePath("game_state")
-                .type(ServerMessage.MessageType.INFO)
-                .jsonData(gson.toJson(gameState))
-                .build();
+        return ServerMessage.builder().messagePath("game_state").type(ServerMessage.MessageType.INFO).jsonData(gson.toJson(gameState)).build();
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason